Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-buster as base
+
+EXPOSE 5050
+WORKDIR /todo-app
+COPY . /todo-app
```

Comment:
This works fine, but note that you could speed up re-builds by COPYing just the poetry files at first, and then copy the rest (or just the todo_app folder) **after** poetry install.

That way you don't need to re-run the relatively slow `pip install poetry... poetry install` layer when all you've changed is app.py for example.